### PR TITLE
SpiceAgent: Send notification when file transfer is complete 

### PR DESCRIPTION
**Pull request by MacDue**
Rather than (re-)open the Downloads folder on every completed file transfer, send a notification when a transfer is complete. The notification includes the file's name, and clicking it will open the Downloads folder (with the file highlighted).
image

### CODE REVIEW

File: Userland/Services/NotificationServer/NotificationWindow.cpp [hunk 1]
```diff
@@ -69,6 +70,11 @@ NotificationWindow::NotificationWindow(i32 client_id, String const& text, String
     set_rect(rect);
 
     auto widget = NotificationServer::NotificationWidget::try_create().release_value_but_fixme_should_propagate_errors();
+    widget->set_greedy_for_hits(true);
```
Comment (nico) [inline Userland/Services/NotificationServer/NotificationWindow.cpp:73]: The close button still closes the notification, right?

**Review by nico — APPROVED** [review]
Spiffy!

Somewhat relatedly, someone mentioned that some of the spice changes causes black qemu windows sometimes: https://discord.com/channels/830522505605283862/830739873119207426/1306728175794917409

File: Userland/Services/NotificationServer/NotificationWindow.cpp (hunk 1 above)
Comment (MacDue) [reply Userland/Services/NotificationServer/NotificationWindow.cpp:73]: Yep, I think that's part of the window (rather than the main widget).

**Comment by MacDue** [comment]
> Somewhat relatedly, someone mentioned that some of the spice changes causes black qemu windows sometimes: https://discord.com/channels/830522505605283862/830739873119207426/1306728175794917409

Works on my machine™ I know very little about Void Linux, but it sounds like maybe GTK is just broken on their system?

**Pull request merged**